### PR TITLE
add contributors and lurkers to readme

### PR DESCRIPTION
**Pull request by Gakkenyak**

### CODE REVIEW

File: README.md
Comment (drikusroor) [inline README.md:132]: It's common practice in markdown to leave a whitespace between a heading and the underlying text

**Review by drikusroor — APPROVED** [review]
LGTM, I'll leave it up to you if you want to improve it or not

**Review by naomi-eliasar — APPROVED** [review]

**Comment by drikusroor** [comment]
Congratulations on your first PR on Github! 🚀  Here's a cat gif to celebrate:

**Review by adrianabm — APPROVED** [review]
LGTM :shipit:

**Pull request merged**
